<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>komponentler</class>
 <widget class="QMainWindow" name="komponentler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elektronik Komponentler</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>buton_resim/anakart.png</normaloff>buton_resim/anakart.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="liste">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>5</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="ekle_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>        Ekle</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/ekle.png</normaloff>buton_resim/ekle.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="sil_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>           Sil</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/sil.png</normaloff>buton_resim/sil.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="guncelle_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>   Güncelle</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/guncelle.png</normaloff>buton_resim/guncelle.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="listele_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>     Listele</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/listele.png</normaloff>buton_resim/listele.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="tur_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tür     :</string>
    </property>
   </widget>
   <widget class="QLabel" name="deger_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Değer :</string>
    </property>
   </widget>
   <widget class="QLabel" name="stok_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stok    :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deger">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="stok">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="tur">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Direnç</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diyot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bobin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Transistör</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kondansatör</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ara_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ara</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/arama.png</normaloff>buton_resim/arama.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="ara">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ozel_tur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tümü</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Direnç</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diyot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bobin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Transistör</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kondansatör</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Listeleme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="temizle_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>    Temizle</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>buton_resim/temizle.png</normaloff>buton_resim/temizle.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
